scenarios/05_The_Swamp_Things.cfg

```diff
@@ -84,15 +87,15 @@
         name=_"Ssumar"
         side=3
         canrecruit=yes
-        {GOLD 100 110 120}
-        {INCOME 1 2 3}
+        {GOLD4 100 110 120 130}
+        {INCOME4 1 2 3 4}
         team_name=bad
         recruit=Saurian Skirmisher,Saurian Augur,Saurian Soothsayer
         [ai]
             recruitment_pattern=scout,scout,healer
             grouping=offensive
-            {QUANTITY aggression 0.4 0.5 0.6}
-            {QUANTITY caution 0.3 0.2 0.1}
+            {QUANTITY4 aggression 0.4 0.5 0.6 0.7}
```

Comment:
Oh by the way I thought I'd mention that just increasing aggression doesn't always make scenarios harder. Sometimes it does the opposite of what you want, just food for thought.

Reply:
well yeah if you let it get above 1.0 it starts making things easier again... I think 0.7 is still fine, though